rust/src/dataset/scanner.rs

```diff
@@ -225,6 +225,7 @@ impl Scanner {
                         manifest.clone(),
                         PREFECTH_SIZE,
                         true,
+                        false,
```

Comment:
I am not sure that how much difference it is before ordered / unordered at the moment. Should we just implement ordered scan, if later we find that there are meat here to change to unordered, by that time, we add this flag later?